Dockerfile

```diff
@@ -1,24 +1,24 @@
-FROM golang:1.11.9
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
 ENV PKG=/go/src/github.com/openshift/osde2e/
+ENV GOMODULE111=on
```

Comment:
We don't need this now, right?

Reply:
You saw nothing

![image](https://user-images.githubusercontent.com/1394552/70250159-7ae47500-174b-11ea-9a19-49c80508e984.png)